src/apps/common/writers.py

```diff
@@ -0,0 +1,64 @@
+def empty_bytearray(preallocate: int) -> bytearray:
+    b = bytearray(preallocate)
```

Comment:
Great rename. Could you maybe add a short note on why we're doing this?

Reply:
Yep!